TFT/src/User/API/config.h

```diff
@@ -7,6 +7,7 @@ extern "C" {
 
 #include <stdbool.h>
 #include "ff.h"
+#include "includes.h"
```

Comment:
I don't see any dependency on "include.h" in config.h. "include.h" is already present in "config.c"